### DISCUSSION TITLE
Brotli compression

### DISCUSSION THREAD

**Discussion by NicoDevriendt** [ChilliCream/hotchocolate#6354] [Q&A]
Hello

I try to connect in a c# application with Strawberryshake a graphql server that compress the response 
when there is a lot of data to return.

With small responses there is no problem, i receive json with my answers on my graphql query

But with big response, there is a brotli compression and StrawberryShake goes in error because there is no json returned.

How is it possible tot handle qraphql queries with brottli compression and StrawberryShake ?

I'm on the version of 12.15 of StrawberryShake 

Thanks

Nico

**Comment by tobias-tengler** [comment]
I haven't tested it yet, but does the following work?

```csharp
services.AddConferenceClient()
    .ConfigureHttpClient(client => ...)
    .ConfigurePrimaryHttpMessageHandler(config => new HttpClientHandler
    {
        AutomaticDecompression = DecompressionMethods.Deflate | DecompressionMethods.Brotli
    });
```

**Reply by NicoDevriendt** [reply]
I haven't the enum value Brotli from DecompressionMethods
So I think for this code I need to upgrade the .NET version 

The version of my System.Net that is necessary to use DecompressionMethods is 
Assembly netstandard, Version=2.0.0.0
